src/Components/EmptyPage/EmptyPage.jsx

```diff
@@ -1,14 +1,14 @@
 import { Link } from "react-router-dom";
 import { Button } from "../index";
 
-export const EmptyPage = ({imageSrc, altText, pageName}) => {
+export const EmptyPage = ({imageSrc, altText, pageText}) => {
```

Comment:
suggestion you can here itself write pageText=''
so you don't have to check at line11